Core.Telemetry/OpenTelemetryLoggerProvider.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public class OpenTelemetryLoggerProvider : ILoggerProvider
+    {
+        private readonly ILoggerFactory loggerFactory;
+
+        public OpenTelemetryLoggerProvider()
+        {
+            switch (OpenTelemetryHelpers.OpenTelemetryExporter)
+            {
+                case OpenTelemetryHelpers.GenevaExporterName:
+
+                    this.loggerFactory = LoggerFactory.Create(builder =>
+                    {
+                        builder.AddOpenTelemetry(loggerOptions =>
+                        {
+                            loggerOptions.IncludeFormattedMessage = true;
+                            loggerOptions.AddGenevaLogExporter(exporterOptions =>
+                            {
+                                exporterOptions.ConnectionString = "EtwSession=OpenTelemetry";
+                            });
+                        });
+                    });
+
+                    break;
+
+                default:
+
+                    this.loggerFactory = LoggerFactory.Create(builder =>
+                    {
+                        builder.AddOpenTelemetry(loggerOptions => loggerOptions.AddConsoleExporter());
+                    });
```

Comment:
I noticed we have the default section for Metrics, Traces,.. below to export/see the logs on the console window. Do we know when would GenevaExporter be incapable of exporting logs to geneva? I'm curious to know what conditions trigger this, Thanks

Reply:
This will be unable to export to Geneva on any environment that does not include a windows monitoring agent. Mostly this is when we run locally but also if we run on the S1 app service tier which does not support MA.